### PR TITLE
delete unncessary args; loading model

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/wing/wing.py [hunk 1]
```diff
@@ -81,15 +72,19 @@ def setup(self, static, state):
...
+            independentvars = [CL, Re]
+        elif fd["d"] == 3:
+            independentvars = [CL, Re, static["\\tau"]]
+
         constraints = [
             Cd >= cdp + CL**2/np.pi/static["AR"]/e,
             Re == state["\\rho"]*state["V"]*static["c_{MAC}"]/state["\\mu"],
             # XfoilFit(fd, cdp, [CL, Re], airfoil="jho1.dat"),
-            XfoilFit(fd, cdp, [CL, Re]),
+            XfoilFit(fd, cdp, independentvars),
```
Comment (bqpd) [inline gpkitmodels/GP/aircraft/wing/wing.py]: I'd rather these be passed in as optional arguments I think....`Cl=, Re=, tau=` etc

**Pull request merged**